README.md

```diff
@@ -1,89 +1,134 @@
 # Hyperdash Python SDK
 
-The Hyperdash Python SDK is the official SDK for [Hyperdash.io](https://hyperdash.io). Once installed, the SDK automatically monitors your machine learning jobs.
+[Hyperdash](https://hyperdash.io) is a machine learning monitoring library capable of running alongside Tensorflow, Scikit-Learn, and other modelling libraries. It was developed with a focus on enabling fast knowledge gain.
 
-## Installation
+Use Hyperdash if you're looking for cloud-based model monitoring that:
 
-`pip install --upgrade pip && pip install hyperdash`
+* Is fast and easy-to-use with scripts and Jupyter.
+* Tracks your hyperparameters across different model experiments.
+* Graphs performance metrics (loss, reward, etc.) in real-time.
+* Can be viewed remotely on [__web__](https://hyperdash.io/dashboard), [__iOS__](https://itunes.apple.com/us/app/hyperdash-machine-learning-monitoring/id1257582233), and [__Android__](https://play.google.com/store/apps/details?id=com.hyperdash) without self-hosting (e.g. Tensorboard).
+* Saves your experiment's print output (standard out / error) as a local log file.
+* Notifies you when a long-running experiment has finished.
 
-## Usage
+Hyperdash is compatible with: **Python 2.7-3.6**
 
-The Hyperdash SDK requires a valid API key in order to function. Luckily, the `hyperdash login` (if you already have an account) and `hyperdash signup` (if you don't) commands will automatically install one for you.
+## Installation
+*Foreword: We care deeply about making Hyperdash fast and easy to install  on Linux, Mac, and Windows. If you find a snag along the way, please let us know at support@hyperdash.io!*
 
-If you'd rather manage your API key manually, then review the "API Key Storage" section below.
+Install Hyperdash in terminal from [pip](https://packaging.python.org/tutorials/installing-packages/#requirements-for-installing-packages):
+```bash
+$ pip install --upgrade pip && pip install hyperdash
+```
+Installing within a python virtual environment such as [virtualenv](https://github.com/pypa/virtualenv) is recommended. If you are having trouble installing via pip, a virtual environment will usually fix the problem.
+```bash
+# Login if you have an account
+$ hyperdash login
 
-### Standalone Script
-The easiest way to use the Hyperdash SDK is to simply prefix any terminal command with `hyperdash run`:
+# Or signup free with an email
+$ hyperdash signup
+```
+After `login` or `signup`, an API key is saved to your local machine for automatic authorization. If you'd rather manage your API key manually, then review the "API Key Storage" section below.
 
+You're ready to use Hyperdash! Make sure Hyperdash works by running:
 ```
-hyperdash run -n "My test python script" python my_test_script.py
+$ hyperdash demo
 ```
 
-or
-
+# Learn Hyperdash in 60 seconds
+## Pure logging 
+If all you need is logging and notifications, simply prefix **any** terminal command:
+```bash
+hd run -n "Hotdog CNN" python hotdog.py
```

Comment:
Are we jumping right to hd? Why are both referenced? Let’s pick one and stick with it.

Reply:
Yeah let's jump to `hd`